utility/importer/chrome_importer.cc

```diff
@@ -484,6 +484,10 @@ void ChromeImporter::ImportPasswords(
     for (auto& entry : blocklist) {
       importer::ImportedPasswordForm form;
       if (PasswordFormToImportedPasswordForm(entry, form)) {
+        if (form.blocked_by_user &&
```

Comment:
perhaps it is more logical to move the check into `PasswordFormToImportedPasswordForm` ?